apps/dev-proxy.config.js

```diff
@@ -1,14 +1,17 @@
 const HOST = process.env.MOLGENIS_APPS_HOST || "https://emx2.dev.molgenis.org";
 const SCHEMA = process.env.MOLGENIS_APPS_SCHEMA || "pet%20store";
 
-const opts = { changeOrigin: true, secure: false };
+const opts = { changeOrigin: true, secure: false, logLevel: "debug" };
```

Comment:
do you want to enable 'debug' logging by default ? , suggest to remove

Reply:
yes, I indeed want to enable debug by default. The amount of messages is limited and meanwhile I had quite a few cases where this has helped to understand why proxy server doesn't operate correctly.